Mode-B patch of multiversioning-clone-only.ll for Julia 1.11 on LLVM 23, generated by test/upstream/derive.jl
from the rules (lines changed): target-features-sign (13)
--- a/multiversioning-clone-only.ll
+++ b/multiversioning-clone-only.ll
@@ -127,7 +127,7 @@
 ; CHECK-DAG: "julia.mv.clones"="2"
 ; CHECK-DAG: "julia.mv.clone"="0"
 ; CHECK-DAG: "target-cpu"="cpubase"
-; CHECK-DAG: "target-features"="nofeatures"
+; CHECK-DAG: "target-features"="-nofeatures"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[DECLARATION_DEFAULT_ATTRS]]
 ; CHECK-SAME: {
@@ -134,7 +134,7 @@
 ; CHECK-DAG: "julia.mv.clones"="2"
 ; CHECK-DAG: "julia.mv.clone"="0"
 ; CHECK-DAG: "target-cpu"="cpubase"
-; CHECK-DAG: "target-features"="nofeatures"
+; CHECK-DAG: "target-features"="-nofeatures"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[SUBTARGET_CLONED_DEFAULT_ATTRS]]
 ; CHECK-SAME: {
@@ -141,7 +141,7 @@
 ; CHECK-DAG: "julia.mv.clones"="6"
 ; CHECK-DAG: "julia.mv.clone"="0"
 ; CHECK-DAG: "target-cpu"="cpubase"
-; CHECK-DAG: "target-features"="nofeatures"
+; CHECK-DAG: "target-features"="-nofeatures"
 ; CHECK-DAG: "julia.mv.reloc"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[CALL_SUBTARGET_CLONED_DEFAULT_ATTRS]]
@@ -149,7 +149,7 @@
 ; CHECK-DAG: "julia.mv.clones"="6"
 ; CHECK-DAG: "julia.mv.clone"="0"
 ; CHECK-DAG: "target-cpu"="cpubase"
-; CHECK-DAG: "target-features"="nofeatures"
+; CHECK-DAG: "target-features"="-nofeatures"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[BORING_CLONEALL_ATTRS]]
 ; CHECK-SAME: {
@@ -156,7 +156,7 @@
 ; CHECK-DAG: "julia.mv.clones"="2"
 ; CHECK-DAG: "julia.mv.clone"="1"
 ; CHECK-DAG: "target-cpu"="cpucloneall"
-; CHECK-DAG: "target-features"="cloneall"
+; CHECK-DAG: "target-features"="-cloneall"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[DECLARATION_CLONEALL_ATTRS]]
 ; CHECK-SAME: {
@@ -163,7 +163,7 @@
 ; CHECK-DAG: "julia.mv.clones"="2"
 ; CHECK-DAG: "julia.mv.clone"="1"
 ; CHECK-DAG: "target-cpu"="cpucloneall"
-; CHECK-DAG: "target-features"="cloneall"
+; CHECK-DAG: "target-features"="-cloneall"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[SUBTARGET_CLONED_CLONEALL_ATTRS]]
 ; CHECK-SAME: {
@@ -170,7 +170,7 @@
 ; CHECK-DAG: "julia.mv.clones"="6"
 ; CHECK-DAG: "julia.mv.clone"="1"
 ; CHECK-DAG: "target-cpu"="cpucloneall"
-; CHECK-DAG: "target-features"="cloneall"
+; CHECK-DAG: "target-features"="-cloneall"
 ; CHECK-DAG: "julia.mv.reloc"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[SUBTARGET_CLONED_FASTMATH_ATTRS]]
@@ -178,7 +178,7 @@
 ; CHECK-DAG: "julia.mv.clones"="6"
 ; CHECK-DAG: "julia.mv.clone"="2"
 ; CHECK-DAG: "target-cpu"="cpufastmath"
-; CHECK-DAG: "target-features"="fastmathclone"
+; CHECK-DAG: "target-features"="-fastmathclone"
 ; CHECK-DAG: "julia.mv.reloc"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[CALL_SUBTARGET_CLONED_CLONEALL_ATTRS]]
@@ -186,7 +186,7 @@
 ; CHECK-DAG: "julia.mv.clones"="6"
 ; CHECK-DAG: "julia.mv.clone"="1"
 ; CHECK-DAG: "target-cpu"="cpucloneall"
-; CHECK-DAG: "target-features"="cloneall"
+; CHECK-DAG: "target-features"="-cloneall"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[CALL_SUBTARGET_CLONED_FASTMATH_ATTRS]]
 ; CHECK-SAME: {
@@ -193,7 +193,7 @@
 ; CHECK-DAG: "julia.mv.clones"="6"
 ; CHECK-DAG: "julia.mv.clone"="2"
 ; CHECK-DAG: "target-cpu"="cpufastmath"
-; CHECK-DAG: "target-features"="fastmathclone"
+; CHECK-DAG: "target-features"="-fastmathclone"
 ; CHECK-SAME: }
 ; CHECK: attributes #[[SUBTARGET_ALIASED_ATTRS]]
 ; CHECK-SAME: {
@@ -210,9 +210,9 @@
 !1 = !{i32 1, !"julia.mv.annotated", i32 1}
 !2 = !{i32 1, !"julia.mv.specs", !3}
 !3 = !{!4, !5, !6, !7, !8}
-!4 = !{!"cpubase", !"nofeatures", i32 0, i32 2}
-!5 = !{!"cpucloneall", !"cloneall", i32 0, i32 2}
-!6 = !{!"cpufastmath", !"fastmathclone", i32 0, i32 4}
+!4 = !{!"cpubase", !"-nofeatures", i32 0, i32 2}
+!5 = !{!"cpucloneall", !"-cloneall", i32 0, i32 2}
+!6 = !{!"cpufastmath", !"-fastmathclone", i32 0, i32 4}
 !7 = !{!"cpuloop", !"loopclone", i32 0, i32 8}
 !8 = !{!"cpusimd", !"simdclone", i32 0, i32 16}
 ; CHECK-DAG: ![[TBAA_CONST_METADATA]] = !{![[JTBAA_CONST_METADATA:[0-9]+]], ![[JTBAA_CONST_METADATA]]
